.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,37 @@
+name: "\U0001F41B Bug Report"
+description: Report a bug on huggingface_hub
+labels: [ "bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a code sample that reproduces the problem you ran into. It can be a Colab link or just a code snippet.
```

In many cases, once users isolate their issue, the issue solves itself.

```suggestion
      description: Please provide a minimal reproducible code which we can copy/paste and reproduce the issue.
```